.gitignore

```diff
@@ -10,3 +10,4 @@
 build
 hostkey.ser
 /bin/
+.DS_Store
```

Comment:
What is this?
I don't mind, but curious what is the point to have such a change :smile: